### DISCUSSION TITLE
Send commande to the Bot

### DISCUSSION THREAD

**Discussion by jpzimmer** [adepierre/Botcraft#71] [Q&A]
I'm trying to use the 2_ChatCommandExample program. The bot displays well but I can't see how to send it a command (I use the no display mode : BOTCRAFT_USE_OPENGL_GUI=OFF and ChatCommandClient client(false) ).
Thanks for your informations.

**Comment by adepierre** [comment]
Available commands should be printed in the console at start (see [this](https://github.com/adepierre/Botcraft/blob/7a0fb025636f06cdcf4d94a2c2534e768a0ac0e8/Examples/2_ChatCommandExample/src/ChatCommandClient.cpp#L21)).

You have to send the commands directly from the minecraft in game chat. For example if you want your bot named "Foo" to switch a lever at pos 45 64 -24:

> Foo interact 45 64 -24

**Reply by jpzimmer** [reply]
Thanks that works well !
